### ISSUE TITLE
AVT 2 - TextArea Placeholder text in form input fields does not meet WCAG 2.0 color contrast requirements

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4187]
Environment
macOS Mojave version 10.14.5
Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
Voice Over

Detailed Description
Use Color Contrast Analyzer to test the background color of the Search placeholder text against the background color of the text input field. See screenshot below:
<img width="834" alt="Screen Shot 2019-10-02 at 3 18 40 PM" src="https://user-images.githubusercontent.com/21676914/66078622-f77a8d80-e527-11e9-8c9f-5594c84016fc.png">

See WCAG Form Tutorial for placeholder text guidance.
Note: All form inputs with placeholder text appear to have this issue.
